### PR TITLE
Prepare v2 (node version set to v0.11.2)

### PR DESCRIPTION
**Pull request by enten**
See #6 and #7

### CODE REVIEW

File: Readme.md [hunk 1]
```diff
@@ -6,11 +6,34 @@
 
 ### Usage
 
+#### `isAbsolute(path)`
```
Comment (mrjoelkemp) [inline Readme.md]: `isRelative`

File: package.json [hunk 2]
```diff
@@ -1,8 +1,11 @@
 {
   "name": "is-relative-path",
-  "version": "1.0.2",
+  "version": "2.0.0",
```
Comment (mrjoelkemp) [inline package.json]: No need. I'll bump via npm when cutting the release.

File: test/test.js [hunk 3]
```diff
@@ -3,21 +3,23 @@ var isRelative = require('../');
 
 describe('is-relative-path', function () {
   it('returns true if the given path is relative', function () {
+    assert.ok(isRelative(''));
```
Comment (mrjoelkemp) [inline test/test.js:6]: That seems weird. What does that resolve to in node?

File: Readme.md (hunk 1 above)
Comment (enten) [reply Readme.md]: Ah! Copy/past mistake

File: test/test.js (hunk 3 above)
Comment (enten) [reply test/test.js:6]: In node's documentation of path.isAbsolute():

> If  the given path is a zero-length string, false will be returned.

I want to verify the complement of that use case.

**Comment by enten** [comment]
I didn't figure why there are dependency problems in the travis' build.

**Comment by mrjoelkemp** [comment]
If you downgrade jscs (https://github.com/mrjoelkemp/is-relative-path/pull/8/files#diff-b9cfc7f2cdf78a7f4b91a753d10865a2R30) in package.json to `^2.x.x`, the issue should go away. Not sure what happened to the cst module, but maybe a broken publish.

**Comment by enten** [comment]
I will make another PR (again) because I made lot of mistakes.

**Pull request closed without merging**